bundles/basic_modes/mode_definitions.moon

```diff
@@ -293,7 +293,8 @@ common_auto_pairs = {
     }
 
   python:
-    extensions: { 'sc', 'py', 'pyw' }
+    extensions: { 'sc', 'py', 'pyw', 'pyx' }
+    patterns: { 'wscript$', 'SConstruct', 'SConscript', 'BUILD' }
```

Comment:
These are too generic. First off, apart from the first one they're not anchored, meaning these will match files such as `RUBYBUILD`, `FILESConstruct`, etc. Even if they were anchored (using `^`), `BUILD` is so generic I wouldn't any mode to claim that.